caab-service/src/test/java/uk/gov/laa/ccms/caab/api/entity/converter/BooleanStringConverterTest.java

```diff
@@ -26,8 +28,23 @@ void testToDomainValue_returnsCorrectResponse(String trueValues, String falseVal
 
     Arrays.stream(trueVals).forEach(s -> assertTrue(booleanStringConverter.toDomainValue(s)));
     Arrays.stream(falseVals).forEach(s -> assertFalse(booleanStringConverter.toDomainValue(s)));
-    assertNull(booleanStringConverter.toDomainValue("unknownval"));
-    assertNull(booleanStringConverter.toDomainValue(null));
+  }
+
+  @Test
+  void testToDomainValue_nullValue_returnsNull() {
+    BooleanStringConverter booleanStringConverter = new BooleanStringConverter(
+        new String[]{"Y"}, new String[]{"N"});
+
+    assertThrows( RuntimeException.class, () -> booleanStringConverter.toDomainValue(
+        "unknownval"));
+  }
+
+  @Test
+  void testToDomainValue_unknownValue_throwsException() {
```

Comment:
It looks like the test names need to be switched, otherwise all good.

Reply:
Corrected :-)